src/main/java/io/cdap/plugin/gcp/bigquery/sink/BigQueryMultiSink.java

```diff
@@ -76,11 +76,13 @@ protected void prepareRunInternal(BatchSinkContext context, BigQuery bigQuery, S
     baseConfiguration.set(BigQueryConstants.CONFIG_OPERATION, Operation.INSERT.name());
     Map<String, String> arguments = new HashMap<>(context.getArguments().asMap());
     FailureCollector collector = context.getFailureCollector();
+    boolean hasMultiSinkPrefix = false;
```

Comment:
Is this change failing pipeline deployment? Doesnt `prepareRunInternal ` get called from `prepareRun` being tat runtime error?

Reply:
That's correct - the error is runtime error. Updated the description to match.